lua/wire/client/pickcolor_browser.lua

```diff
@@ -0,0 +1,109 @@
+--[[
```

Comment:
`/lua/wire/client/text_editor` is still an appropriate place for text-editor-specific E2-specific stuff (see `/lua/wire/client/text_editor/modes/e2.lua`).